README.md

```diff
@@ -1,100 +1,63 @@
+A powerful language integrated query (LINQ) library for Go.
+* Safe for concurrent use
+* Complete lazy evaluation
+* Supports arrays, slices, maps, strings, channels and
+custom collections (collection needs to implement Iterable interface
+and element - Comparable interface)
+* Parallel LINQ (PLINQ) *(comming soon)*
 
-[home]: http://ahmetalpbalkan.github.io/go-linq/
-
-A powerful language integrated query library for Go. Inspired by Microsoft's
-[LINQ](http://msdn.microsoft.com/en-us/library/bb397926.aspx).
+## Installation
 
-* **No dependencies:** written in vanilla Go!
-* **Tested:** 100.0% code coverage on all stable [releases](https://github.com/ahmetalpbalkan/go-linq/releases).
-* **Backwards compatibility:** Your integration with the library will not be broken
-  except major releases.
+    $ go get gopkg.in/ahmetalpbalkan/go-linq.v2
 
-## Installation
+## Quickstart
 
-    $ go get github.com/ahmetalpbalkan/go-linq
+Usage is as easy as chaining methods like
 
-## Quick Start
+`From(slice)` `.Where(predicate)` `.Select(selector)` `.Union(data)` 
 
-**Example query:** Find names of students over 18:
+Just keep writing.
 
+**Example:** Find all owners of cars manufactured from 2015
 ```go
-import . "github.com/ahmetalpbalkan/go-linq"
+import . "gopkg.in/ahmetalpbalkan/go-linq.v2"
 	
-type Student struct {
-    id, age int
-    name string
+type Car struct {
+    id, year int
+    owner, model string
 }
 
-over18Names, err := From(students)
-	.Where(func (s T) (bool,error){
-		return s.(*Student).age >= 18, nil
-	})
-	.Select(func (s T) (T,error){
-		return s.(*Student).name, nil
-	}).Results()
-```
-
-## Documentation
-
-* [GoDoc at godoc.org](http://godoc.org/github.com/ahmetalpbalkan/go-linq)
-* [GoDoc at codebot.io](http://codebot.io/doc/pkg/github.com/ahmetalpbalkan/go-linq)
-
-Here is wiki:
-
-1. [Install & Import](https://github.com/ahmetalpbalkan/go-linq/wiki/Install-&-Import)
-2. [Quickstart (Crash Course)](https://github.com/ahmetalpbalkan/go-linq/wiki/Quickstart)
-3. [Parallel LINQ][plinq]
-4. [Table of Query Functions](https://github.com/ahmetalpbalkan/go-linq/wiki/Query-Functions)
-5. [Remarks & Notes](https://github.com/ahmetalpbalkan/go-linq/wiki/Remarks-%26-notes)
-6. [FAQ](https://github.com/ahmetalpbalkan/go-linq/wiki/FAQ)
-
-[plinq]: https://github.com/ahmetalpbalkan/go-linq/wiki/Parallel-LINQ-(PLINQ)
-
-## License
-
-This software is distributed under Apache 2.0 License (see [LICENSE](LICENSE)
-for more).
-
-## Disclaimer
-
-As noted in LICENSE, this library is distributed on an "as is" basis and
-author's employment association with Microsoft does not imply any sort of
-warranty or official representation the company. This is purely a personal side
-project developed on spare times.
+owners := []string{}
 
-## Authors
-
-[Ahmet Alp Balkan](http://ahmetalpbalkan.com) – [@ahmetalpbalkan](https://twitter.com/ahmetalpbalkan)
-
-## Release Notes
```

Comment:
Hmm you can leave it here. I was planning to add a big notice to the top about the breaking change after merging this.